index.js

```diff
@@ -0,0 +1,36 @@
+const express = require("express");
+const cors = require("cors");
+const bcrypt = require("bcryptjs");
+
+const db = require("./data/dbConfig.js");
+const PORT = 4200;
+
+const server = express();
+
+server.use(express.json(), cors());
+
+// | POST   | /api/register | Creates a `user` using the information sent inside the `body` of the request. Hashes the password
+
+server.post("./api/register", (req, res) => {
+  const user = req.body;
+  user.password = bcrypt.hashSync(user.password, 12);
```

Comment:
It may be tedious, but I recommend adding a server side check to be sure that the user provided a username and a password.